src/components/metadata/StaticMetafields.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import StaticMetaField from './statics/StaticMetaField';
+import Icon from '../Icon';
+
+export class StaticMetaFields extends Component {
```

Comment:
These all can be a function component since they won't have lifecycle methods

Reply:
Of course..!! :man_facepalming: 
I forgot about that..